.grabl/automation.yml

```diff
@@ -42,52 +42,27 @@ build:
         sudo unlink /usr/bin/python3
         sudo ln -s $(which python3) /usr/bin/python3
         sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
-        ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME \
-        ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD \
-        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
```

Comment:
`netrc` credentials are not required to download Grakn because it is a public artifact.